### DISCUSSION TITLE
Example 15 SphinXsys site

### DISCUSSION THREAD

**Discussion by Mirzaei97** [Xiangyu-Hu/SPHinXsys#470] [Ideas]
Hello
I downloaded the Sphinxsys program from your site but the sample 15 did not build and run on the site and did not give us the output.
Also, this example is not available in the output of the tests, which can be seen at the following address
https://github.com/Xiangyu-Hu/SPHinXsys/tree/master/tests/2d_examples
Can you guide me?

**Comment by Xiangyu-Hu** [comment]
Thanks for try SPHinXsys. What is the name of example 15?

**Comment by Mirzaei97** [comment]
Thank you for your reply.
Example 15: 2D aortic valve
Link to this example:
https://www.sphinxsys.org/html/examples/example15_2D_aortic_valve.html

**Comment by Xiangyu-Hu** [comment]
Sorry for inconvenience. 
This test case is not included in the master branch but as a problem for my teaching on bio-fluid mechanics.
@DongWuTUM Could you handle this issue? Thanks.

**Reply by Mirzaei97** [reply]
Thank you for your answer

**Comment by DongWuTUM** [comment]
@Mirzaei97 Hello, thank you for your interest in SPHinXsys. Regarding the aortic valve, all the source code is available on the website you provided: https://www.sphinxsys.org/html/examples/example15_2D_aortic_valve.html. You can try modifying certain codes based on 'test_2d_fsi2' to make the aortic valve case run on the new version of SPHinXsys.

One feasible approach is to duplicate the entire 'test_2d_fsi2' folder, rename the folder and files accordingly, then paste the source code for the aortic valve from the website. Afterward, update the code according to 'test_2d_fsi2.' If you have any further questions, feel free to contact me.

**Reply by Mirzaei97** [reply]
Thank you for your guidance
I will try it

**Reply by Xiangyu-Hu** [reply]
I would prefer to compile the aortic valve case directly and solve all the compiling error and try again.

**Comment by Mirzaei97** [comment]
Dear @DongWuTUM

I hope all is fine. I was going to run example 15 ( 2D aortic valve) in Sphinxsys and show the result in Paraview. Still, unfortunately, there isn't any output file in Sphinxsys, I shared this issue with you, and you gave me a solution ( approach is to duplicate the entire 'test_2d_fsi2' folder, rename the folder and files accordingly, then paste the source code for the aortic valve from the website)for it, did it and when I ran it but it doesn't work for me. The displayed error is in the attached 
PDF file.
I look forward to hearing from you

Thanks and with best regards
Mirzaei
[error pdf.pdf](https://github.com/Xiangyu-Hu/SPHinXsys/files/13639011/error.pdf.pdf)

**Reply by DongWuTUM** [reply]
It shows the case cannot compile. You can learn the test_2d_fsi2 case, and make the aortic case compile first.
